packages/compass-components/src/hooks/use-confirmation.tsx

```diff
@@ -129,7 +130,7 @@ export const ConfirmationModalArea: React.FC = ({ children }) => {
     <ConfirmationModalContext.Provider value={contextValue}>
       {children}
       <ConfirmationModal
-        data-testid="confirmation-modal"
+        data-testid={confirmationProps.dataTestId ?? 'confirmation-modal'}
```

Comment:
Yea. I just noticed in old e2e tests that we were fine using the default testid. It will probably be best to not complicate the interface here. I am going to apply the suggesting here.